src/beanmachine/ppl/diagnostics/tools/utils/model_serializers.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""Collection of serializers for the diagnostics tool use."""
+
+from typing import Dict, List
+
+from beanmachine.ppl.inference.monte_carlo_samples import MonteCarloSamples
+
+
+def serialize_bm(samples: MonteCarloSamples) -> Dict[str, List[List[float]]]:
```

Comment:
again, possible to use InferenceData instead of MCS?

Reply:
It is, however, I think we should leave this as a future feature request since these tools are pretty feature loaded as is right now.